fastlane/lib/fastlane/helper/adb_helper.rb

```diff
@@ -24,6 +24,10 @@ def initialize(adb_path: nil, adb_host: nil)
           adb_path = File.join(android_home, "platform-tools", "adb")
         end
 
+        if adb_path.start_with?('~')
```

Comment:
Why only if the path starts with? Shouldn't `File.expand_path` be safe all the time?

Reply:
I agree, should be safe all the time. I just wanted to do the changes as minimal-invasive as possible since I don't know the code base well and want to make sure my PR gets merged. Updated the code accordingly.

Reply:
Seems you were not wrong ;)

Reply:
Reverted then, again. Especially since `shellescape` only affects paths starting with `~` (not those starting with `.`) so I think this was the correct solution for that reason as well. :)